core/misc/material/add.svg

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
```

Comment:
Elsewhere in core we are avoiding svg icons, due to concerns about older browser support. Where we do use them we have to use them as an override and also support the png fallback (see the bottom of modules/system/css/messages.theme.css for an example).

It would be simpler to switch back to .pngs like these:
* http://fa2png.io/r/font-awesome/plus/?color=878787&margin=0&size=64 
* http://fa2png.io/r/font-awesome/minus/?color=878787&margin=0&size=64